includes/acl/algorithm/uniformly_sampled/common.h

```diff
@@ -49,7 +49,7 @@ namespace acl
 				PtrOffset32<uint8_t>	track_data_offset;
 			};
 
-			struct FullPrecisionHeader
+			struct Header
```

Comment:
Rename this to ClipHeader to avoid confusion with other headers (e.g. SegmentHeader)